pkg/object/httpserver/listen.go

```diff
@@ -78,7 +84,7 @@ func (l *LimitListener) SetMaxConnection(n uint32) {
 // Close closes LimitListener.
```

Comment:
Comment for the line 81: Do we need to monitor the returning channel of `l.sem.SetMaxCount` or not? If not, please eliminate it.

Reply:
It is not monitored currently.
But I propose to keep it as `l.sem.SetMaxCount` is an async operation, and `sem` is a utility package, it may be useful later.

Reply:
If so, please comment it because I got confused.

Reply:
updated.

Reply:
I agree with keeping this `done` chan, cause in some scenarios, the user may want to make sure the acquirement is done successfully.